resources/shaders/raytracing.hlsl

```diff
@@ -266,8 +268,9 @@ void ClosestHitEntry(inout HitInfo payload, in MyAttributes attr)
 		s0,
 		uv);
 
-	float3 albedo = output_data.albedo;
-	float roughness = output_data.roughness;
+	float3 albedo = pow(output_data.albedo, 2.2f);
+  //float3 albedo = output_data.albedo;
+  float roughness = output_data.roughness;
```

Comment:
There seem to be a lot of inconsistant tabbing problems in all of your changes, double check if they are formatted correctly or if this is just the web viewer giving these weird spacings.